### PR TITLE
add info about setting for Edge > 77

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8409

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by illfated** [comment]
If you look at the originating page, https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-smartscreen/microsoft-defender-smartscreen-available-settings, it shows the same issue as in the Rich Diff view of the PR content (https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8542/files?short_path=12e67b3#diff-12e67b38e445922ed975beb4b144adf92ff9d4bd71fa12a0626dfe435bb0cb59), that row 1 and 2 shows the Description content moved to a 4th column, whereas the rest of the table only shows 3 columns.

Would it be practical & feasible to remove the cell divider between column 2 and 3 in row 1 and 2? (I can offer to make the suggestions, if you don't see how to change it yourself.)

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - Copy looks good but please see @illfated's comment.

**Comment by VLG17** [comment]
> al & feasible to remove the cell divider between column 2 and 3

@illfated Thanks for looking into this. I'd really appreciate if you could make the suggested changes, so I don't try and do something and not be ok and double the work. Next time i'll know what to look after.

**Comment by illfated** [comment]
You are more than welcome. Thanks for the good words. By the way, it looks like I will have to make a PR from my own fork to your PR branch, because those 2 extra columns are out of reach for the changes in this PR.

**Comment by illfated** [comment]
There you go. I forgot to add the line "Updates #8542" in the commit part, that is why you only see the PR itself linked here. Never mind, it is not a big deal.

**Comment by illfated** [comment]
See commit https://github.com/MicrosoftDocs/windows-itpro-docs/commit/f248872c9b4861a1d92ea8a945899fe6280fcab2 (`microsoft-defender-smartscreen-available-settings.md` file link https://github.com/MicrosoftDocs/windows-itpro-docs/commit/f248872c9b4861a1d92ea8a945899fe6280fcab2#diff-12e67b38e445922ed975beb4b144adf92ff9d4bd71fa12a0626dfe435bb0cb59) for the file conflict details.

Feel free to choose between resolving those 3 changes (single space, comma, and word insert) yourself, or tell me if you would like to have them suggested in the PR content instead.

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you @illfated.

**Comment by VLG17** [comment]
> See commit [f248872](https://github.com/MicrosoftDocs/windows-itpro-docs/commit/f248872c9b4861a1d92ea8a945899fe6280fcab2) (`microsoft-defender-smartscreen-available-settings.md` file link [f248872#diff-12e67b38e445922ed975beb4b144adf92ff9d4bd71fa12a0626dfe435bb0cb59](https://github.com/MicrosoftDocs/windows-itpro-docs/commit/f248872c9b4861a1d92ea8a945899fe6280fcab2#diff-12e67b38e445922ed975beb4b144adf92ff9d4bd71fa12a0626dfe435bb0cb59)) for the file conflict details.
> 
> Feel free to choose between resolving those 3 changes (single space, comma, and word insert) yourself, or tell me if you would like to have them suggested in the PR content instead.

@illfated Could you please suggest them in the PR content? I'm not confortable with changing what is needed and the probability of doing something wrong is quite high, which would just add to the total time invested in a minor edit.

**Comment by illfated** [comment]
Very well, fair enough. I'll do my best, it should be ready within the next 30-45 minutes (unless something crashes).

**Comment by illfated** [comment]
There you go. It should be as close as possible to the public branch content.

**Comment by illfated** [comment]
Hmm. Looks like there must be some remaining traces. I will have to compare the files in an online diff view to see what differs between them still. There seems to be a line number discrepancy by at least 1, have to check what goes on there.

**Comment by illfated** [comment]
Well, I copied the raw contents of both files into Notepad++ and looked at the diff there. To me, it looks like there is only the added changes in this PR, but could be mistaken.

**Comment by illfated** [comment]
If you find it too difficult to edit the conflict here from the GitHub page, you could consider opening a new PR with the same content, just making sure that the file you edit is the updated one as of after October 27, 2020.

**Comment by VLG17** [comment]
> If you find it too difficult to edit the conflict here from the GitHub page, you could consider opening a new PR with the same content, just making sure that the file you edit is the updated one as of after October 27, 2020.

I think i did it...i hope

**Comment by illfated** [comment]
Well, you seem to have pulled it off, because GitHub does not complain about any file conflict anymore. Good job, I say. 👍🏻

File: windows/security/threat-protection/microsoft-defender-smartscreen/microsoft-defender-smartscreen-available-settings.md [hunk 1]
```diff
@@ -34,28 +34,31 @@ SmartScreen uses registry-based Administrative Template policy settings. For mor
...
+<b>At least Windows Server 2012, Windows 8 or Windows RT</b></td>
 <td>This policy setting turns on Microsoft Defender SmartScreen.<p>If you enable this setting, it turns on Microsoft Defender SmartScreen and your employees are unable to turn it off. Additionally, when enabling this feature, you must also pick whether Microsoft Defender SmartScreen should Warn your employees or Warn and prevent bypassing the message (effectively blocking the employee from the site).<p>If you disable this setting, it turns off Microsoft Defender SmartScreen and your employees are unable to turn it on.<p>If you don't configure this setting, your employees can decide whether to use Microsoft Defender SmartScreen.</td>
 </tr>
 <tr>
 <td><b>Windows 10, version 2004:</b><br>Administrative Templates\Windows Components\Windows Defender SmartScreen\Explorer\Configure App Install Control</td>
 <td><b>Windows 10, version 1703:</b><br>Administrative Templates\Windows Components\Windows Defender SmartScreen\Explorer\Configure App Install Control</td>
+patch-107
+=======
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-smartscreen/microsoft-defender-smartscreen-available-settings.md:45]: This text is a left-over from the conflict resolution process.
```suggestion
```

File: windows/security/threat-protection/microsoft-defender-smartscreen/microsoft-defender-smartscreen-available-settings.md [hunk 2]
```diff
@@ -34,28 +34,31 @@ SmartScreen uses registry-based Administrative Template policy settings. For mor
...
 <td><b>Windows 10, version 1703</td>
+public
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-smartscreen/microsoft-defender-smartscreen-available-settings.md:47]: Correction: This is a combination of text remnants from the conflict resolution process and the one of the column cells we want to remove:
```suggestion
```

**Review by Bahaayaser — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Nice. This PR is finally complete again. Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**